company-profiles/basisPointDesignLtd.md

```diff
@@ -0,0 +1,25 @@
+# basispointdesignltd
+
+## Company blurb
+
+Basis Point Design is a brand, web and marketing communications specialist that services emerging and established investment managers across the globe.
+
+## Company size
+
+11-50 employees
+
+## Remote status
+
+Branding, Graphic Design, Marketing Presentations, Website Design, Website Development, Marketing Communications, Investor Communications, Pitch Deck Design, Digital Design, Investor Portal Design and Build, Marketing Emailers, Video Production, Animation, Infographics, UI UX Design, Infographics, 3D Motion, Marketing Videos, Audio Pitch Decks, Fact Sheet Design, and Investor Reporting
+
+## Region
+
+England
+
+## Company technologies
+
+WordPress, JavaScript, CSS, SCSS, HTML,Next.js, Typescript, Node.js, Vue.js, React.js, Gatsby.js,PHP, Laravel, MySql
+
+## How to apply
+
+Visit https://basispoint.design/
```

Comment:
I don't see any mention about open careers or remote employment. Can you add context, please?